tree.go

```diff
@@ -258,4 +258,5 @@ var (
 	xfdf    = newMIME("application/vnd.adobe.xfdf", ".xfdf", magic.Xfdf)
 	glb     = newMIME("model/gltf-binary", ".glb", magic.Glb)
 	jxr     = newMIME("image/jxr", ".jxr", magic.Jxr).alias("image/vnd.ms-photo")
+	parquet = newMIME("application/vnd.apache.parquet", ".parquet", magic.Par1)
```

Comment:
`application/vnd.apache.parquet` was standardised at IANA a few months ago. `application/x-parquet` was the pseudo-standard before that and some software might still be using the old mime, ex: https://docs.aws.amazon.com/sagemaker/latest/dg/clarify-processing-job-data-format-tabular.html#clarify-processing-job-data-format-tabular-prereq

To accommodate for older software let's add `application/x-parquet` as alias.
```suggestion
	parquet = newMIME("application/vnd.apache.parquet", ".parquet", magic.Par1).
		alias("application/x-parquet")
```

Reply:
Branch updated to include `application/x-parquet` alias.